### DISCUSSION TITLE
Why do the tutorials use stderr() instead of stdout()?

### DISCUSSION THREAD

**Discussion by ghost** [ratatui-org/website#119] [Q&A] [answered]
Every app I've seen uses stdout(), not stderr(). I'm not saying it's wrong I'm just wondering why this is.

**Comment by kdheepak** [comment, marked as answer]
This is covered in this section of the FAQ:

https://ratatui.rs/faq/index.html#should-i-use-stdout-or-stderr

Open to PRs to improve the documentation!

**Reply by ghost** [reply]
Thanks, missed it.

**Reply by kdheepak** [reply]
It’s a fairly new section!
